tests/Tokenizer/TokensTest.php

```diff
@@ -772,6 +772,17 @@ public function testFindBlockEndInvalidStart()
         $tokens->findBlockEnd(Tokens::BLOCK_TYPE_DYNAMIC_VAR_BRACE, 0);
     }
 
+    /**
+     * @expectedDeprecation Argument #3 of Tokens::findBlockEnd is deprecated and will be removed in 3.0, use Tokens::findBlockStart or drop the argument.
+     * @group legacy
+     */
+    public function testFindBlockEndLastParameterDeprecated()
+    {
+        $tokens = Tokens::fromCode('<?php ${$bar};');
+
+        $tokens->findBlockEnd(Tokens::BLOCK_TYPE_DYNAMIC_VAR_BRACE, 4, false);
```

Comment:
You should still check the result of this call as it must continue to work even if deprecated. You should also check that the deprecation is triggered if the third argument is passed `true`.